CLA.md

```diff
@@ -1 +1,38 @@
-# Contributor License Agreement
\ No newline at end of file
+# Contributor License Agreement
+
+Project: Beyond Nations
+
+Parties:
+
+Contributor
+Project Maintainer: Fairfield Programming Association ("FPA")
+Third Party: Preponderous Software ("Preponderous")
```

Comment:
Since Preponderous Software isn't a legal entity yet, I want this changed to my full name: "Daniel McCoy Stephenson"

Further, I want to be defined as a "Project Maintainer", the same as the FPA, and I would prefer that my name go first.